GitUI/Editor/FileViewer.cs

```diff
@@ -1500,15 +1499,7 @@ public void ResetNoncommittedSelectedLines()
                 { currentItemStaged, "--reverse --index" }
             };
 
-            string output = Module.GitExecutable.GetOutput(args, patch, throwOnErrorOutput: false);
-            if (EnvUtils.RunningOnWindows())
-            {
-                // remove file mode warnings
-                Regex regEx = new("warning: .*has type .* expected .*", RegexOptions.Compiled);
```

Comment:
❔ Have you removed `RegexOptions.Compiled` on purpose when moving it below?

Reply:
Yes, that should increase the startup time a lot more than decrease the runtime.
Most regex are static already, plan to look in to that later.

This was from #9468